npm/private/npm_translate_lock_generate.bzl

```diff
@@ -98,7 +98,7 @@ def generate_repository_files(rctx, pnpm_lock_label, importers, packages, patche
 
     npm_imports = helpers.get_npm_imports(importers, packages, patched_dependencies, only_built_dependencies, root_package, rctx.name, rctx.attr, rctx.attr.lifecycle_hooks, rctx.attr.lifecycle_hooks_execution_requirements, rctx.attr.lifecycle_hooks_use_default_shell_env, npm_registries, default_registry, npm_auth)
 
-    link_packages = [helpers.link_package(root_package, import_path) for import_path in importers.keys()]
+    link_packages = sorted([helpers.link_package(root_package, import_path) for import_path in importers.keys()])
```

Comment:
starlark spec says "Iteration yields the dictionary's keys in the order in which they were inserted; updating the value associated with an existing key does not affect the iteration order." so not sure this is necessary?

Reply:
This did change the generated code, however maybe the `importers` should be sorted elsewhere? Or maybe it is deterministic (lockfile order) and there is no reason for this one. I might revert this line...